README.md

```diff
@@ -53,3 +53,26 @@ openshift-install destroy cluster
 Note that you almost certainly also want to clean up the installer state files too, including `auth/`, `terraform.tfstate`, etc.
 The best thing to do is always pass the `--dir` argument to `install` and `destroy`.
 And if you want to reinstall from scratch, `rm -rf` the asset directory beforehand.
+
+
+### Troubleshouting installation
+
+```Cleanup``` step *should* tear down everything created during the previous installation, however sometimes the installer may report error messages like the ones below:
```

Comment:
@wking either ```destroy``` step was interrupted, or stopped due to network issue, but I do not have clear reproducer. When I wanted to install new cluster it was not possible due to  error described here, but possible after applying fix described here ... thanks to @dobbymoodge for suggestions

Reply:
> ... either `destroy` step was interrupted, or stopped due to network issue...

There are lots of ways this could go wrong.  The most reliable approach to cleaning it up should be to re-launch `destroy`.  Can you post logs showing that a fresh `destroy` call failed to delete the profile or other resource?  I'm not even clear on how you were able to run a new `create cluster` without having previously had a successful `destroy cluster` on the previous cluster.  What version of the installer were you running?  0.10.1 has #1039, with some `destroy cluster` fixes.

Reply:
q:  Can you post logs showing that a fresh destroy call failed to delete the profile or other resource? 
a: unfortunately I do not have it. 
prior to every install I do git pull, ./hack/build.sh - so I was on latest.